client/app/models/package.js

```diff
@@ -67,21 +67,14 @@ export default class PackageModel extends Model {
 
   async save() {
     await this.fileManager.save();
-<<<<<<< HEAD
     if (this.dcpPackagetype === DCPPACKAGETYPE.PAS_PACKAGE.code) {
-=======
```

Comment:
nit:  should move this deletion to an earlier commit, just for ease of commit review and history cleanliness